core/spec/models/spree/variant_spec.rb

```diff
@@ -240,6 +241,29 @@
     it 'should order by bar than foo' do
       expect(variant.options_text).to eql 'Bar Type: Bar, Foo Type: Foo'
     end
+
+  end
+
+  describe 'exchange_name' do
+    let!(:variant) { create(:variant, option_values: []) }
+    let!(:master) { create(:master_variant) }
+
+    before do
+      variant.option_values << create(:option_value, {name: 'Foo', presentation: 'Foo', option_type: create(:option_type, position: 2, name: 'Foo Type', presentation: 'Foo Type')})
```

Comment:
Line is too long. [180/100]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.